### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by shiqimei** [boardx/boardx-core#43] [Announcements]
We are excited to release our first alpha version `v0.1.0` with following features delivered:
 
 - 🎉 Basic Boards Management (Create, Rename, Switch etc.)
 - 🎉 Favorite Boards (thanks to @CulturalProfessor )
 - 🎉 Boards Search Feature (thanks to @CulturalProfessor )

<hr /><em>This discussion was created from the release <a href='https://github.com/markboard-io/markboard/releases/tag/'>v0.1.0</a>.</em>

**Comment by CulturalProfessor** [comment]
> We are excited to release our first alpha version `v0.1.0` with following features delivered:
> 
>     * tada Basic Boards Management (Create, Rename, Switch etc.)
> 
>     * tada Favorite Boards (thanks to @CulturalProfessor )
> 
>     * tada Boards Search Feature (thanks to @CulturalProfessor )
> 
> 
> _This discussion was created from the release [v0.1.0](https://github.com/markboard-io/markboard/releases/tag/)._

All thanks to your guidance :raised_hands:
